### ISSUE TITLE
Import 350Mb Gedcom file

### ISSUE THREAD

**Issue by faustzar** [fisharebest/webtrees#1776]
Struggling to import a 350Mb Gedcom file, is it possible, I have set the total memory_limit in php to 2Gb, get to 20% then fail the import with out of memory issue, any advice is appreciated.

**Comment by fisharebest** [comment]
The import process is designed to work with large files and servers with low CPU/memory limits.

It divides the file into chunks of 64KB
It spends a maximum of one second loading chunks, before starting a new process.

It would seem more likely that your GEDCOM file contains some "problematic" data which causes an infinite loop.

Look in the table wt_gedcom_chunk for the first entry that has `imported = 0`.

Is there anything "unusual" here?

Are you able to let me have a copy of the GEDCOM file so that I can investigate on my local development machine?  I can probably find the cause very quickly if I can reproduce it locally.

**Comment by faustzar** [comment]
I can, should I email? (zip is 65mb)

**Comment by fisharebest** [comment]
My email address is fisharebest@gmail.com

If it is too big for mail, try sharing it via google drive?

**Comment by fisharebest** [comment]
If you still need my help, then ask on the forum or open another issue on github.
